src/main/java/com/baidu/hugegraph/manager/BackupManager.java

```diff
@@ -281,6 +294,10 @@ private void backup(HugeType type, int number, List<?> list) {
         }
     }
 
+    private String initPage() {
+        return this.backend.equals(MEMORY) ? null : "";
```

Comment:
can keep it consistent?

Reply:
memory is special due to not support paging